config-cell-cleanup.html.md.erb

```diff
@@ -39,36 +39,14 @@ Selecting the best option for disk cleanup depends on the workload that the Dieg
 
 For PCF installations that primarily run buildpack-based apps, Pivotal recommends selecting the **Routinely clean up Cell Disk space** option. The **Routinely clean up Cell Disk space** option ensures that when a new stack becomes available on a cell, the old stack is dropped immediately from the cache.
 
-For PCF installations that primarily run Docker images, or both Docker images and buildpack-based apps, Pivotal recommends selecting the **Clean up Disk space once a threshold is reached** option with a reasonable threshold.
+For PCF installations that primarily run Docker images, or both Docker images and buildpack-based apps, Pivotal recommends selecting the **Clean up Disk space once usage fills disk** option with a reasonable reserve.
 
-### <a id='choosing-a-threshold'></a> Calculating a Threshold
+### <a id='choosing-a-reserve'></a> Calculating a Reserve
 
-To calculate a realistic value when configuring the disk cleanup threshold, you must identify some of the most frequently used Docker images in your PCF installation. Docker images tend to be constructed by creating layers over existing base images. In some cases, you may find it easier to identify which base Docker images are most frequently used.  
-
-To calculate the disk cleanup threshold, do the following:
-
-1. Identify the most frequently used Docker images or base Docker images. 
-
-1. Using the Docker CLI, measure the size of those images.
-<br><br>
-For example, you may determine the most frequently used images in a test deployment are `openjdk:7`, `nginx:1.13`, and `php:7-apache`. In this case, you can run the following commands to pull the identified images locally, then measure their sizes:
-<pre class="terminal">
-$> docker pull openjdk:7
-$> docker pull nginx:1.13
-$> docker pull php:7-apache
-<br><br>
-$> docker images
-REPOSITORY       TAG          IMAGE ID          CREATED             SIZE
-php              7-apache     2720c02fc079      2 days ago          391 MB
-openjdk          7            f45207c01009      5 days ago          586 MB
-nginx            1.13         3448f27c273f      5 days ago          109 MB
-...
-</pre>
-
-1. Calculate the threshold as the sum of the frequently used image sizes plus a 15-20% buffer.
-<br><br>
-For example, using the output above, the sample threshold calculation is: (&nbsp;391&nbsp;MB&nbsp;+&nbsp;586&nbsp;MB&nbsp;+&nbsp;109&nbsp;MB&nbsp;)&nbsp;*&nbsp;1.2&nbsp;=&nbsp;1303.2&nbsp;MB
+The space reserved for other jobs will be set to a sensible default in the case that the only other component to consider is the diego-cache.
```

Comment:
hey-- is there an actual default value pre-populated in this PAS field? if so, can we explicitly mention it? and is the implication that if you are running other jobs/components on the Cell, you should consider increasing the reserve (could this apply to the case of Small Footprint PAS?)

Reply:
We set it to a sensible default and minimum for pcf installations and we already account for the other items in /var/vcap/data (which is the same for small-footprint and vanilla diego-cells). It's 15GB. It seems odd to mention it as it will pre-populate automatically with this value.

We don't account for any space needed for bosh-addons that need storage, however we have left about 1GB of buffer.

Reply:
thanks @mrosecrance! we should mention the default here since the UI does not mention it and it's possible an operator may change it, want to change it back later, but then cannot remember what the default was. Or maybe could even forget whether they set it or not.